ynput/OpenPype
# Global: custom location for OP local versions

## Changelog Description
This provides configurable location to unzip Openpype version zips. By default, it was hardcoded to artist's app data folder, which might be problematic/slow with roaming profiles.

Location must be accessible by user running OP Tray with write permissions (so `Program Files` might be problematic)

## Additional info
It would be beneficial to allow artist to change it for themselves, but I am not sure if it makes sense to invest time to add it to Local Settings with Ayon impending. 

## Testing notes:
1. configure `Custom Local Versions Folder` in System Settings
2. provide zip to `Versions repository`
3. check that zip was consumed and unzipped in location configured in step 1
